new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java

```diff
@@ -0,0 +1,391 @@
+   * 2) Verify that the WebLogic domain whose namespace = op-wls-ns comes up.
+   */
+  @Test
+  @Order(2)
+  @DisplayName("Set dedicated to true and verify that the domain deployed in the operator's namespace comes up")
+  public void testDedicatedModeSameNamespace() {
+    // create and verify the domain
+    logger.info("Creating and verifying model in image domain");
+    createDomain(domain1Namespace);
+    verifyDomainRunning(domain1Namespace);
+  }
+
+  /**
+   * Scale up cluster-1 in domain1Namespace and verify it succeeds.
+   */
+  @Test
+  @Order(3)
+  @DisplayName("Scale up cluster-1 in domain1Namespace and verify it succeeds")
+  public void testDedicatedModeSameNamespaceScale() {
+    // scale the cluster and check domain can be managed from the operator
+    int externalRestHttpsPort = getServiceNodePort(opNamespace, "external-weblogic-operator-svc");
+    logger.info("externalRestHttpsPort {0}", externalRestHttpsPort);
+
+    logger.info("scaling the cluster from {0} servers to {1} servers", replicaCount, replicaCount + 1);
+    scaleAndVerifyCluster(clusterName, domainUid, domain1Namespace,
+        managedServerPodPrefix, replicaCount, replicaCount + 1,
+        true, externalRestHttpsPort, opNamespace, opNamespace + "-sa",
+        false, "", "", 0, "",
+        "", null, null);
+  }
+
+  /**
+   * Test when a CRD with a lower than expected version is present,
+   * Operator fails with error if it has not permission to overwrite the CRD.
+   */
+  @Test
+  @Order(4)
+  @Disabled("Disable the test because the Operator has permission to overwrite the CRD")
+  @DisplayName("Create a CRD with a lower than expected version and verify that Operator fails with error")
+  public void testDedicatedModeNlowerVersionCrd() {
+    // delete existing CRD
+    new Command()
+        .withParams(new CommandParams()
+            .command("kubectl delete crd domains.weblogic.oracle --ignore-not-found"))
+        .execute();
+
+    // install a lower version of CRD, v2.6.0
+    new Command()
+        .withParams(new CommandParams()
+            .command("kubectl create -f " + ITTESTS_DIR + "/../kubernetes/crd/domain-v1beta1-crdv7-260.yaml"))
+        .execute();
+
+    try {
+      // install latest version of operator and verify
+      logger.info("Installing and verifying operator");
+      installAndVerifyOperator(opNamespace, opNamespace + "-sa",
+          false, 0, opHelmParams, domainNamespaceSelectionStrategy,
+          false, domain2Namespace);
+
+      // we expect installAndVerifyOperator fails with a lower than expected version of CRD
+      fail("Installing the Operator should fail with a lower than expected version of CRD");
+    } catch (Exception ex) {
+      logger.info("Installing the Operator with a lower than expected version of CRD failed as expected");
+    } finally {
+      // restore the test env
+      uninstallOperatorAndVerify();
+    }
+  }
+
+  /**
```

Comment:
More Description needed.

Reply:
That's exactly use case. I don't know what else to give :(